ods_ci/tests/Tests/0500__ide/0501__ide_jupyterhub/custom-image.robot

```diff
@@ -94,9 +93,11 @@ Test Bad Image URL
 
 Test Image From Local registry
     [Documentation]  Try creating a custom image using a local registry URL (i.e. OOTB image)
-    ...       Issue reported for this test in the past - https://github.com/opendatahub-io/odh-dashboard/issues/2185
     [Tags]    Tier1
     ...       ODS-2470
+    ...       ExcludeOnDisconnected    # Since we don't have internal image registry enabled there usually
+    ...       InternalImageRegistry    # Requires internal image registry enabled
+    ...       ProductBug               # https://issues.redhat.com/browse/RHOAIENG-1193
```

Comment:
If the test has `ProductBug`, it will be run only in the known-issues quality gate. Beware RHOAIENG-1193 is closed as "Won't Do"

Reply:
Yeah, TBH, I was wondering whether we shouldn't remove the test at all... will consider it and maybe raise a separate issue.